tests/precalculations/test_SorensenDicePrecalculation.py

```diff
@@ -12,14 +12,16 @@ def setUp(self):
                                           'ab cd ef', 'ab cd ef', 'ab cd ef', 'ab cd ef', 'ab cd ef',
                                           'ab cd ef', 'ab cd ef', 'ab cd ef', 'ab cd ef', 'ab cd ef',
                                           'ab cd ef', 'ab cd ef', 'ab cd ef', 'ab cd ef', 'ab cd ef']})
-        self.df2 = pd.DataFrame({'col1': ['ab ','hi ','jk ','lm ','no ','pq ','rs ','tu ','vw ','xy ','z1 ','23 ','45 ','67 ','89 ']})
+        self.df2 = pd.DataFrame({'col1': ['ab ', 'hij', 'jkl', 'lmn', 'nop', 'pqr', 'rst', 'tuv',
+                                          'vwx', 'xyz', 'z12', '234', '456', '678', '890', 'zyx',
+                                          'xwv', 'vut', 'tsr', 'rqp']})
 
         self.store = Store(self.df1, self.df2)
         self.result = SorensenDicePrecalculations(ngram_type=NGramType.character, n=3).process(self.store)
 
     def test_result(self):
         self.assertEqual(len(self.result), 1)
-        self.assertEqual(self.result['col1'], (1.0, 0.0, (2 / 15) / 7))   # ngrams get normalized during join
+        self.assertEqual(self.result['col1'], (1.0, 0.0, (2 / 20) / 7))   # ngrams get normalized during join
```

Comment:
Switch actual and expected

Reply:
What do you mean with that comment? Since I changed the size of the test data, this change was intended